### ISSUE TITLE
lr scheduler

### ISSUE THREAD

**Issue by msafari18** [MAGICS-LAB/DNABERT_2#150]
Hi, I am trying to pretrain a model on DNABERT-2, but the model's loss is not converging. I wanted to as,k did you use this scheduler for learning rate? 
composer.optim.LinearWithWarmupScheduler
